### ISSUE TITLE
Project License?

### ISSUE THREAD

**Issue by phikai** [gitpod-io/self-hosted#20]
How is this project licensed?

How is GitPod self-hosted licensed for use?

**Comment by VerifiedGruber** [comment]
This remains an open question. I'd guess it follows Gitlab's "Open Core" with [a MIT License](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/LICENSE) for the open-sourced part of the code.

**Comment by wulfthimm** [comment]
That is correct. You do need a license if you want to use private repositories, for example.
